hgvs/normalizer.py

```diff
@@ -155,6 +155,14 @@ def normalize(self, var):
                 ref_end = end - 1
                 edit = hgvs.edit.NARefAlt(ref=ref, alt=alt)
 
+        # ensure the start is not 0
+        if ref_start == 0:
```

Comment:
Yes. 3' shifting could result in insertions outside of the end of reference sequence, for example 'NM_212556.2:c.1400_1401insAC'.

I just updated this pull request to handle this issue.